modules/core/src/v2/keychains.ts

```diff
@@ -1,16 +1,24 @@
 import common = require('../common');
 import * as _ from 'lodash';
 import * as Promise from 'bluebird';
+import { BaseCoin } from './baseCoin';
+import { NodeCallback } from './types';
+import { Wallet } from './wallet'
 const util = require('../util');
 const co = Promise.coroutine;
 
-const Keychains = function(bitgo, baseCoin) {
+export interface KeyPair {
+  pub?: string;
+  prv: string;
+}
+
+const Keychains = function(bitgo: any, baseCoin: BaseCoin) {
   this.bitgo = bitgo;
   this.baseCoin = baseCoin;
 };
 
-Keychains.prototype.get = function(params, callback) {
-  params = params || {};
+Keychains.prototype.get = function(params: { id: string, xpub?: string, ethAddress?: string, reqId?: string }, callback: NodeCallback<any>) {
```

Comment:
Please pull out into interface named `GetKeychainOptions`. The usual naming pattern for options interfaces is <methodname><classname>Options

Reply:
Needs return type as well.